packages/compute-baseline/src/baseline/index.ts

```diff
@@ -90,6 +90,11 @@ export function computeBaseline(
   },
   compat: Compat = defaultCompat,
 ): SupportDetails {
+  // findKeystoneDate is used here because it returns the latest date from an
+  // array of releases. The result isn't a keystone date of any feature.
+  const cutoffDate = findKeystoneDate(browsers(compat).map((b) => b.current()));
+  assert(cutoffDate !== null);
+
```

Comment:
This adds a new condition to the Baseline high date: nothing can reach Baseline high until 30 months after Baseline low _and_ at least one browser has released on or since that date.

If the aim here is to pin ourselves to BCD's state then I'd like to suggest some alternatives:

- BCD's `__meta.timestamp` value (e.g., we can never get ahead of BCD's release schedule)
- The last recorded release date in BCD, even if it's a future release date (something like `browsers(compat).map((b) => b.releases.at(-1))`).

Reply:
I've gone with `__meta.timestamp`. The latest release in the releases array can be a planned release months into the future, so it's not a good approximation of now. Currently, Edge and Firefox have planned releases in September in BCD.